src/coreclr/tools/Common/Compiler/DisplayNameHelpers.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System;
+using System.Collections.Generic;
```

Comment:
Nit - left over from the changed to this - file - just revert the file to the original.